lib/generate-critical-css.js

```diff
@@ -8,12 +10,15 @@ async function generateCriticalCSS( {
 	urls,
 	viewports,
 	filters,
+	meta,
```

Comment:
Instead of `meta.required_valid_urls_number` and `meta.max_urls_number`, I would prefer more contextually meaningful variable names.

How about:
```
successTargets: { min: 5, max: 20 },
```

i.e.: I want at least 5 successes, but stop after 20 successes

What do you think?

Reply:
@thingalon, thanks, I was struggling the naming, these makes sense. I did some renaming in this repo in commit c5a9fac81a55e7b3d47f506b78faea4e9163bbcd and in https://github.com/Automattic/jetpack-boost/pull/960/commits/7841160a46275d32c29b6341d42184b42a718714